File: Corredor/Digitales/C#/C#/Estudiante.cs
namespace EscuelaMusica
{
    public class Estudiante
    {
        public string Nombre { get; set; }
        public string Instrumento { get; set; }
        public int Nivel { get; set; }

        public Estudiante(string nombre, string instrumento, int nivel)
        {
            Nombre = nombre;
            Instrumento = instrumento;
            Nivel = nivel;
        }

        public void Practicar()
        {
            System.Console.WriteLine($"{Nombre} practica {Instrumento} (nivel {Nivel}).");
        }
    }
}

File: Corredor/Digitales/JavaScript/Estudiante.js
class Estudiante {
    constructor(nombre, instrumento, nivel) {
        this.nombre = nombre;
        this.instrumento = instrumento;
        this.nivel = nivel;
    }

    practicar() {
        console.log(`${this.nombre} practica ${this.instrumento} (nivel ${this.nivel}).`);
    }
}

module.exports = Estudiante;